src/main/java/org/mule/module/apikit/Configuration.java

```diff
@@ -117,12 +118,20 @@ public void initialise() throws InitialisationException {
     } catch (final Exception e) {
       throw new InitialisationException(e.fillInStackTrace(), this);
     }
-    flowFinder = new FlowFinder(ramlHandler, getName(), locator, flowMappings.getFlowMappings(),
-                                errorRepositoryFrom(muleContext));
+    initFlowFinder();
     buildResourcePatternCaches();
     registry.registerConfiguration(this);
   }
 
+  private void initFlowFinder() {
+    try {
+      flowFinder = new FlowFinder(ramlHandler, getName(), locator, flowMappings.getFlowMappings(),
+                                  errorRepositoryFrom(muleContext));
+    } catch (UnsupportedMediaTypeException e) {
+      throw new UnsupportedOperationException(e.getMessage());
```

Comment:
Is there any reason why it is changed for an `UnsupportedOperationException`. Is this throwing a `415 Unsupported Media Type`? I would add an MTF test for this case.